python/lsst/meas/algorithms/setPrimaryFlags.py

```diff
@@ -80,7 +80,7 @@ def __init__(self, *, schema, isSingleFrame=False, **kwargs):
             doc=primaryDoc,
         )
 
-        if "deblend_nChild" in schema.getNames() or "deblend_scarletFlux" in schema.getNames():
+        if "deblend_nChild" in schema.getNames():
```

Comment:
I gather `deblend_scarletFlux` is still a valid column since it appears elsewhere on this PR; is it just no longer relevant for this logic branch?